blockchain/mod.go

```diff
@@ -38,6 +38,10 @@ type Validator interface {
 
 // Actor is a primitive created by the blockchain to propose new blocks.
 type Actor interface {
+	// InitChain creates a new chain with data as the payload and the players as
+	// the first roster.
+	InitChain(data proto.Message, players mino.Players) error
```

Comment:
Is it creating a first block with "data" as the payload of this first block? If so we should clarify it.